build/images/base/build.sh

```diff
@@ -113,15 +114,24 @@ docker build $PLATFORM_ARG --target cni-binaries \
        --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION \
        --build-arg OVS_VERSION=$OVS_VERSION .
 
+docker build $PLATFORM_ARG --target whereabouts-bin \
```

Comment:
Docker pull/push changes not verified yet (TODO: Discuss with Antonin). 
did local build for antrea/base-ubuntu:2.14.0 and made sure that the docker image gets packaged with whereabouts IPAM bin at /opt/cni/bin/ location.

docker run antrea/base-ubuntu:latest ls -ltr /opt/cni/bin
total 52364
-rwxr-xr-x 1 root root  4246019 Aug 26  2020 bandwidth
-rwxr-xr-x 1 root root  3979034 Aug 26  2020 portmap
-rwxr-xr-x 1 root root  3566204 Aug 26  2020 loopback
-rwxr-xr-x 1 root root  3745368 Aug 26  2020 host-local
-rwxr-xr-x 1 root root 38074904 May 14 23:23 whereabouts